src/ontology/mondo-edit.obo

```diff
@@ -544549,6 +544549,21 @@ intersection_of: MONDO:0000358 ! orofacial cleft
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25839 ! GRHL3
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8457" xsd:anyURI
 
+[Term]
+id: MONDO:0100581
+name: ACTN2-related cardiac and skeletal myopathy
+def: "A congenital myopathy in which the cardiac and skeletal system are affected and the cause of the disease is a mutation in the ACTN2 gene. ACTN2 was first reported in association with autosomal dominant hypertrophic cardiomyopathy in humans in 2006, dilated cardiomyopathy in 2010, left ventricular noncompaction in 2014, restrictive cardiomyopathy in 2016, and left-dominant arrhythmogenic cardiomyopathy in 2020. ACTN2 was also reported in association with walking difficulties and distal as well as proximal muscle weakness in 2019. ACTN2 was also found to cause recessive myopathy where it could also be a founder for variant patients with Palestinian ethnicity in 2024. [PMID:17097056, PMID:20474083, PMID:25224718, PMID:27662471, PMID:31956495, PMID:38311799, PMID:30900782, https://orcid.org/0009-0001-3369-6513]" []
+synonym: "ACTN2 familial isolated dilated cardiomyopathy" NARROW [PMID:17097056, PMID:20474083, PMID:25224718, PMID:27662471, PMID:30900782, PMID:31956495, PMID:38311799]
+synonym: "cardiomyopathy, dilated, 1AA, with or without LVNC" NARROW [PMID:17097056, PMID:20474083, PMID:25224718, PMID:27662471, PMID:30900782, PMID:31956495, PMID:38311799]
+synonym: "cardiomyopathy, hypertrophic, 23, with or without LVNC" NARROW [PMID:17097056, PMID:20474083, PMID:25224718, PMID:27662471, PMID:30900782, PMID:31956495, PMID:38311799]
+synonym: "CMD1AA" NARROW ABBREVIATION [PMID:17097056, PMID:20474083, PMID:25224718, PMID:27662471, PMID:30900782, PMID:31956495, PMID:38311799]
+synonym: "dilated cardiomyopathy 1AA with or without left ventricular noncompaction" NARROW [PMID:17097056, PMID:20474083, PMID:25224718, PMID:27662471, PMID:30900782, PMID:31956495, PMID:38311799]
+synonym: "dilated cardiomyopathy type 1AA" NARROW [PMID:17097056, PMID:20474083, PMID:25224718, PMID:27662471, PMID:30900782, PMID:31956495, PMID:38311799]
+synonym: "familial isolated dilated cardiomyopathy caused by mutation in ACTN2" NARROW [PMID:17097056, PMID:20474083, PMID:25224718, PMID:27662471, PMID:30900782, PMID:31956495, PMID:38311799]
+is_a: MONDO:0019952 {source="PMID:17097056", source="PMID:20474083", source="PMID:25224718", source="PMID:27662471", source="PMID:30900782", source="PMID:31956495", source="PMID:38311799"} ! congenital myopathy
```

Comment:
Why are these synonyms narrow?

Reply:
they overlap with 'dilated cardiomyopathy 1AA' - they are the same synonyms as on this term. Do you think narrow is appropriate or maybe related?

Reply:
yes, narrow is appropriate I think.